utils/m0reportbug

```diff
@@ -645,6 +645,12 @@ backtraces_cores() {
             cp $core $outdir/$IN_DIR/
         fi
     done
+    if [ -d $outdir ] && ! rmdir $outdir 2>/dev/null; then
+        find "$outdir" -type f \( -name core\*.xz -o -name core\*.gz \) | while read dir; do
+            ## Core dumps were collected. We need to collect binaries.
+            COLLECT_BINARIES_P=1
```

Comment:
Why it is needed in case of core dumps, we can use internal repos for the release builds, this needs to be done through command 
line option only?